src/main/java/FridayExceptions/EmptyListException.java

```diff
@@ -0,0 +1,4 @@
+package FridayExceptions;
+
+public class EmptyListException extends Exception{
```

Comment:
Perhaps a space before the bracket, similarly for the rest as well:
```suggestion
public class EmptyListException extends Exception {
```